crates/parcel_core/src/types/asset.rs

```diff
@@ -150,9 +155,20 @@ impl Asset {
   }
 }
 
+impl Hash for Asset {
```

Comment:
Are we unable to derive?

Reply:
I removed this as we no longer need it to be hashable